app/controllers/static_controller.rb

```diff
@@ -7,6 +7,9 @@ def home
   def campaigns
   end
 
+  def mapathon
```

Comment:
we should follow the convention and use the plural form here as well. "mapathons". And as the entity is actually called "Events" in our database, I'd rename this to "events".